serving-api/src/main/scala/com/stratio/sparkta/serving/api/actor/StreamingActor.scala

```diff
@@ -112,40 +109,11 @@ class StreamingActor(streamingContextService: StreamingContextService,
     }
   }
 
```

Comment:
"private def doDeleteContext(contextName: String): Unit = {"

Please, don't remove this function, is necessary for the statuses of the contexts.

In Cluster mode we add the funtionality in the next iterations.